Sources/KIF/KIFAccessibilityEnabler.m

```diff
@@ -20,7 +20,7 @@ - (NSNumber *)AXInspectorEnabled:(id)specifier;
 @end
 
 
-static void * loadDylibForSimulator(NSString *path)
+void * loadDylibForSimulator(NSString *path)
```

Comment:
why did this change?

Reply:
to use it from UIAutomationHelper.m, reverted in https://github.com/openium/KIF/commit/4f286cf6f0264c1b93b5b1f860f00ca0bfb43104